_posts/2019-03-01-editorial-reatures.markdown

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: The Anatomy of an Editorial Feature
+date: 2019-03-01
+author: eve
+categories: [reaction, javascript, publishing]
+---
+
+On select occasions since 2015, Artsy Editorial has created a number of custom, one-off articles featuring unique layouts, styles and experiences. After trying a number of implementations, the [`EditorialFeature`](https://github.com/artsy/reaction/tree/master/src/Components/Publishing/EditorialFeature) component was introduced to the process during Artsy’s 2018 year-in-review projects.  
+
+By moving the implementation of custom articles to Artsy’s component library, we were able to remove some of the friction and time investment necessary for engineers to spin up these articles, and enable bespoke layouts to be housed in Artsy.net’s Article domain rather than a custom Express app. Acting essentially as a wrapper to accept article data, any component can be rendered as a child of the `EditorialFeature` component, allowing for flexible combinations of new and existing features, and for minimal or maximal interventions.
+
+<!-- more -->
+
+For a light-weight customization, a developer might add props for unique text or background colors. Medium-touch could involve wrapping an existing layout in a styled-component declaring further css interventions to font-size, borders, margins or other layout properties. The space is an open canvas, so the option is available to build every element from scratch, introduce JS interactivity, and to interact with any data saved to the article model in a completely new way. The scale of a project can vary widely, but determined by weighing priorities of editorial intent, proposed designs, engineering capabilities/availability, and budget.
+
+Some examples of articles created with the `EditorialFeature` component include:
+
+- The Most Influential Artists of 2018 - [Components](https://github.com/artsy/reaction/tree/master/src/Components/Publishing/EditorialFeature/Components/Eoy2018Artists) | [Article](https://www.artsy.net/article/artsy-editorial-influential-artists-2018)
+- The Year in Culture 2018 - [Components](https://github.com/artsy/reaction/blob/master/src/Components/Publishing/EditorialFeature/Components/Eoy2018Culture.tsx) | [Article](https://github.com/artsy/reaction/tree/master/src/Components/Publishing/EditorialFeature)
```

Comment:
Is there a link to ` The Year in Culture 2018 `? This is cool, I hadn't see one just above until now.

Reply:
Yes! good catch, will add that